easybuild/tools/systemtools.py

```diff
@@ -574,6 +574,33 @@ def get_cpu_features():
     return cpu_feat
 
 
+def get_gpu_info():
+    """
+    Get the GPU info
+    """
+    gpu_info = {}
+    os_type = get_os_type()
+
+    if os_type == LINUX:
+        try:
+            cmd = "nvidia-smi --query-gpu=gpu_name,driver_version --format=csv,noheader"
```

Comment:
Maybe check first whether `nvidia-smi` command is available (via `which` function from `filetools`)?

Reply:
We are inside a `try` / `except` block, so that will catch and report `nvidia-smi` being unavailable.

Reply:
Yeah, ok